kubernetes/windows/Dockerfile

```diff
@@ -31,6 +31,7 @@ RUN refreshenv \
 && gem install gyoku -v 1.3.1 \
 && gem install ipaddress -v 0.8.3 \
 && gem install jwt -v 2.7.1 \
+&& gem uninstall rexml -v 3.2.5 --force \

```

Comment:
Did you verify that after exec into the windows container, rexml ruby gem doesnt exist?

Reply:
yes please check the screenshot